demo/birth-date-picker-demo.tsx

```diff
@@ -0,0 +1,28 @@
+import React = require("react");
+import {BirthDatePicker} from "../src";
+
+export const initialValue = new Date("2001-09-11Z");
+export const minDate = new Date("1900-06-06Z");
+export const maxDate = new Date();
+
+export class BirthDatePickerDemo extends React.Component<{}, {value: Date}> {
+    constructor() {
+        super();
+        this.state = {value: initialValue};
+    }
+
+    handleChange = (newValue: Date) => {
+        this.setState({value: newValue});
```

Comment:
Let's make the demo richer. Let's update some label with the value of the picker. The purpose of the demo is to document, to show how stuff works. It's an important part of the component documentation.